# HG changeset patch
# User Andrew Cooper <andrew.cooper3@citrix.com>
# Date 1394471951 0
# Node ID feb5d135a95a18f1fea5a93160c079e64a2ad294
# Parent  b7157d6646f18ba4cdbd874a7299081db4ae092d
tools/xcutils: Free xtl loggers after use

While not much of a functional change, this prevents irritating warnings from
valgrind when trying to analyse xc_domain_{save,restore}() themselves.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
CC: Ian Campbell <Ian.Campbell@citrix.com>
Acked-by: Ian Jackson <Ian.Jackson@eu.citrix.com>

diff -r b7157d6646f1 -r feb5d135a95a tools/xcutils/xc_restore.c
--- a/tools/xcutils/xc_restore.c
+++ b/tools/xcutils/xc_restore.c
@@ -67,6 +67,7 @@ main(int argc, char **argv)
     }
 
     xc_interface_close(xch);
+    xtl_logger_destroy(l);
 
     return ret;
 }
diff -r b7157d6646f1 -r feb5d135a95a tools/xcutils/xc_save.c
--- a/tools/xcutils/xc_save.c
+++ b/tools/xcutils/xc_save.c
@@ -222,6 +222,7 @@ main(int argc, char **argv)
         xc_evtchn_close(si.xce);
 
     xc_interface_close(si.xch);
+    xtl_logger_destroy(l);
 
     return ret;
 }
